packages/client/scripts/generate-domain-keys.ts

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env node
+
+import { secp256k1 } from '@noble/curves/secp256k1';
+import { randomBytes } from '@noble/hashes/utils';
+import { mkdirSync, writeFileSync } from 'fs';
+import { join } from 'path';
+
+// Base64URL encoding function
+function base64url(buffer: Uint8Array): string {
+  return Buffer.from(buffer)
+    .toString('base64')
+    .replace(/\+/g, '-')
+    .replace(/\//g, '_')
+    .replace(/=/g, '');
+}
+
+function getOutputDir(): string {
+  // Check for --cwd argument
+  const cwdArgIndex = process.argv.indexOf('--cwd');
+  if (cwdArgIndex !== -1 && process.argv[cwdArgIndex + 1]) {
+    return process.argv[cwdArgIndex + 1];
+  }
+  // Fallbacks
+  return process.env.INIT_CWD || process.cwd();
+}
+
+function generateDomainVerificationKeys() {
+  // Generate a random private key
+  const privateKeyBytes = randomBytes(32);
+  const privateKey = base64url(privateKeyBytes);
+  
+  // Get the uncompressed public key from the private key
+  const publicKey = secp256k1.getPublicKey(privateKeyBytes, false); // false = uncompressed
+  
+  // Extract x and y coordinates from the public key
+  // secp256k1 uncompressed public key is 65 bytes: 1 byte prefix + 32 bytes x + 32 bytes y
+  const x = base64url(publicKey.slice(1, 33));
+  const y = base64url(publicKey.slice(33));
+  
+  // Create the JWK
+  const jwk = {
+    kty: 'EC',
+    crv: 'secp256k1',
+    x,
+    y,
+    use: 'sig',
+    kid: 'coinbase-domain-verification',
+    alg: 'ES256K'
+  };
+  
+  // Create the JWKS
+  const jwks = {
+    version: "1.0",
+    keys: [jwk]
+  };
+  
+  return { jwks, privateKey };
+}
+
+function main() {
+  try {
+    const outputDir = getOutputDir();
+
+    console.log('🔑 Generating Coinbase domain verification keys...\n');
+    
+    const { jwks, privateKey } = generateDomainVerificationKeys();
+    
+    // Create .well-known directory if it doesn't exist
+    const wellKnownDir = join(outputDir, '.well-known');
+    mkdirSync(wellKnownDir, { recursive: true });
```

Comment:
I don't think we need to output it to a local directory called .well-known. also its currently but created as a hidden directory which might be annoying